packages/utils/src/npm.ts

```diff
@@ -30,17 +30,25 @@ const install = (command: string) => {
     });
 };
 
+const hasYarnLock = (directory: string): Promise<boolean> => {
```

Comment:
What about using [`promisify`](https://nodejs.org/dist/latest-v10.x/docs/api/util.html#util_util_promisify_original)  to do this?

```ts
import { promisify } from 'util';

const accessAsync = promisify(fs.access);

//...

packageManagerChoice = (await accessAsync(path.join(currentWorkingDir, 'yarn.lock'))) ? 'yarn' : 'npm';
```

Reply:
@molant `promisify` returns Pomise<void> which can't be used to check for truthiness. 
![image](https://user-images.githubusercontent.com/10586637/57246941-287b9280-705c-11e9-9a9e-c236dcb3ad45.png)



Also, the same approach is used in `extension-browser` [here](https://github.com/webhintio/hint/blob/60624562af11362cf834f1791c6f3c1dfe84385d/packages/extension-vscode/src/server.ts#L29).

Reply:
We have other things to worry about, let's leave it as it is 😊